test/common/network/BUILD

```diff
@@ -100,7 +100,6 @@ envoy_cc_test(
 envoy_cc_test(
     name = "dns_impl_test",
     srcs = ["dns_impl_test.cc"],
```

Comment:
Can you add some explicit unit test for this in `dns_impl_test.cc` as well? Thanks.

Reply:
yeah sure thing!